src/gui/touchscreengui.cpp

```diff
@@ -793,6 +793,7 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 		}
 
 		m_pointer_pos[event.TouchInput.ID] = touch_pos;
+		m_pointer_downpos[event.TouchInput.ID] = touch_pos;
```

Comment:
Can these lines be swapped? `m_pointer_downpos` is declared first in the header file.